docker-compose.yaml

```diff
@@ -91,12 +91,10 @@ services:
       - "traefik.enable=true"
       - "traefik.default.protocol=http"
       - "traefik.frontend.rule=Host:api.ethvm.${DOMAIN}"
-      - "traefik.frontend.headers.customResponseHeaders=Access-Control-Allow-Origin:http://ethvm.${DOMAIN}||Access-Control-Allow-Credentials:true"
```

Comment:
Why is this line removed?

Reply:
Hey i was using it instead of Cors, so i can use Chrome. I will put it back when done :)

Reply:
This is what I am using      
 - 'traefik.frontend.headers.customResponseHeaders=Access-Control-Allow-Origin:http://localhost:8080||Access-Control-Allow-Credentials:true'

Reply:
Sorry for gazilion messages, in develop its:  - 'traefik.frontend.headers.customResponseHeaders=Access-Control-Allow-Origin:*||Access-Control-Allow-Credentials:true'
Can you please put the correct one. Not sure which one to use * or http://ethvm.${DOMAIN}